R/qc_log2_intensity_distribution.R

```diff
@@ -0,0 +1,36 @@
+#' Check Log2 intensity distribution
+#'
+#' Plots the overall distribution of all peptide intensities
+#'
+#' @param data A dataframe containing at least sample names, peptide or precursor identifiers and log2 transformed intensities for each peptide or precursor.
+#' @param sample The column in the data dataframe containing the sample name.
+#' @param grouping The column in the data dataframe containing either precursor or peptide identifiers.
+#' @param log2intensity The column in the data dataframe containing the log2 transformed intensities of each precursor or peptide.
+#'
+#' @return A plot that shows the intensity distribution over all samples
+#' @import dplyr
+#' @import ggplot2
+#' @importFrom magrittr %>%
+#' @export
+#'
+#' @examples
+#' \dontrun{
+#' qc_log2_distribution(data,
+#' sample,
+#' grouping,
+#' log2intensity)
+#' }
+qc_log2_intensity_distribution <-
```

Comment:
maybe only call it intensity distribution. The intensities should be log2 transformed to make a nice curve shape but it is also possible to use it with non-transformed intensities.
I would write in the header that it is recommended to be used with log2 transformed intensities to get a bell shaped curve (which you already kind of did).